Source/Python/WarpXWrappers.h

```diff
@@ -37,9 +37,9 @@ extern "C" {
 
     void amrex_init (int argc, char* argv[]);
 
-#ifdef BL_USE_MPI
+    #ifdef BL_USE_MPI
```

Comment:
I know it's old code, but the new variable is called:
```suggestion
#ifdef AMREX_USE_MPI
```

these days :)